From 093b6d79694fde0656f97fe6eacf331d9109f028 Mon Sep 17 00:00:00 2001
From: lazarh <def_58@abv.bg>
Date: Tue, 10 Dec 2024 16:56:05 +0000
Subject: [PATCH] Add support for Cubieboard 4 board with Allwinner 80 SoC from
 the sun9i family.

Configure U-Boot environment location, size (CONFIG_ENV_SIZE) and
offset (CONFIG_ENV_OFFSET) on the FAT partition for Cubieboard4
board and after that use it in /etc/fw_env.config.

Upstream-Status: Inappropriate [RAUC specific]

Signed-off-by: Lazar Hristov <lhristov@gmail.com>
---
 configs/Cubieboard4_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/Cubieboard4_defconfig b/configs/Cubieboard4_defconfig
index d3678ad5c7..2a4accb372 100644
--- a/configs/Cubieboard4_defconfig
+++ b/configs/Cubieboard4_defconfig
@@ -13,3 +13,6 @@ CONFIG_AXP_GPIO=y
 CONFIG_SYS_I2C_SUN8I_RSB=y
 CONFIG_REGULATOR_AXP_USB_POWER=y
 CONFIG_AXP809_POWER=y
+CONFIG_ENV_SIZE=0x20000
+CONFIG_ENV_OFFSET=0x0000
+CONFIG_ENV_IS_IN_FAT=y
-- 
2.47.0

